superchain/implementations/networks/mainnet.yaml

```diff
@@ -0,0 +1,7 @@
+l1_cross_domain_messenger:
+l1_erc721_bridge:
+l1_standard_bridge:
+l2_output_oracle:
+optimism_mintable_erc20_factory:
+optimism_portal:
+system_config:
```

Comment:
Should these be set to the older currently deployed addresses?

Reply:
I left them empty because the current addresses are technically not superchain compatible. Open to adding them, ideally we can have tooling that can grab from storage each of the impl addresses and generate these files given the proxy addresses